openvdb/openvdb/python/pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[build-system]
+requires = [
+    "scikit_build_core",
+    "pybind11",
+    "numpy"
```

Comment:
If OpenVDB uses the Numpy C-API, you might want to take a look at https://github.com/scipy/oldest-supported-numpy to avoid incompatibilities with newer versions and older versions.

Reply:
Excellent idea - I'll make this change.